traffic_portal/app/src/common/modules/form/user/FormUserController.js

```diff
@@ -37,6 +37,16 @@ var FormUserController = function(user, $scope, $location, formUtils, stringUtil
             });
     };
 
+    $scope.changeLogs = [];
+
+    $scope.getChangeLogs = function(userName) {
+        changeLogService.getChangeLogs({ username: userName })
```

Comment:
I believe this call to `getChangeLogs` is un-needed. The first thing the [change-logs page does](https://github.com/apache/trafficcontrol/blob/5b6db34a8a2c930aa56ec73249161f26cd876328/traffic_portal/app/src/modules/private/changeLogs/list/index.js#L35) is run a `getChangeLogs`.

Reply:
oh.. so change the html button to a link instead of a call?

Reply:
That's what @ocket8888 is requesting. If it were to stay a call  the function would just need to be `locationUtils.navigateToPath('/change-logs?username='+userName)`.